tests/sentry/models/test_groupsubscription.py

```diff
@@ -46,17 +46,24 @@ def test_bulk(self):
             user = self.create_user()
             user_ids.append(user.id)
 
-        GroupSubscription.objects.bulk_subscribe(group=group, user_ids=user_ids)
+        teams = []
+        for i in range(20):
```

Comment:
we can use `_` for variables we don't actually use

```suggestion
        for _ in range(20):
```